.github/workflows/release-app.yml

```diff
@@ -0,0 +1,165 @@
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # Creating the tag requires having the whole history of `main`
+          fetch-depth: 0
+
+      - name: Validate `sha-tag` input
+        uses: actions/github-script@v6
+        with:
+          script: |
+            let exists = undefined,
+              page = 0
+            while (!exists) {
+              page += 1
+              const { data: versions } =
+                await github.rest.packages.getAllPackageVersionsForPackageOwnedByOrg({
+                  package_type: 'container',
+                  // We do not have to validate that auxiliary images also exist as they're built at the same time
+                  // as the "main" image. e.g., `api_nginx` is always built when `api` is built and they'll have
+                  // the same set of tags.
+                  package_name: 'openverse-${{ inputs.app }}',
+                  org: 'WordPress',
+                  page,
+                  // max of `per_page`
+                  per_page: 100,
+                })
+              if (!versions.length) {
+                break
+              }
+              exists = versions.some((v) => v.metadata.container.tags.includes('${{ inputs.image-sha }}'))
+            }
+            if (!exists) {
+              throw new Error(
+                `'${{ inputs.image-sha }}' does not appear to be a valid SHA tag for ${{ inputs.app }}.`
+              )
+            }
+
+      - name: Calculate tag name
+        id: tag
+        run: |
+          # Format example: 2023.03.22.04.56.29
+          # `-u` forces UTC
+          formatted_date="$(date -u +%Y.%m.%d.%H.%M.%S)"
+
+          # Split image and git tag to avoid app name duplicated in the fully qualified image name
+          {
+            echo "date=$formatted_date";
+            echo "git-tag=${{ inputs.app }}-$formatted_date";
+            echo "image-tag=rel-$formatted_date";
+          } >> "$GITHUB_OUTPUT"
+
+      - name: Log in to GitHub Docker Registry
+        uses: docker/login-action@v2
+        with:
+          registry: https://ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Add new tag to existing docker image
+        run: |
+          docker buildx imagetools create ghcr.io/wordpress/openverse-${{ inputs.app }}:${{ inputs.image-sha }} --tag ghcr.io/wordpress/openverse-${{ inputs.app }}:${{ steps.tag.outputs.image-tag }}
+
+          if [ "${{ inputs.app}}" = "api" ]; then
+            docker buildx imagetools create ghcr.io/wordpress/openverse-api_nginx:${{ inputs.image-sha }} --tag ghcr.io/wordpress/openverse-api_nginx:${{ steps.tag.outputs.image-tag }}
+          fi
+
+      - name: Create and publish release
+        uses: release-drafter/release-drafter@v5
+        id: release-drafter
+        with:
+          config-name: release-drafter-${{ inputs.app }}.yml
```

Comment:
I don't know if it is possible. The documentation says that it should be in the `.github` folder: https://github.com/release-drafter/release-drafter#action-inputs

I tried to look in the code to see if it would support directories and it's a bit of a rabbit hole that I didn't try to find the end of. If we wanted to keep looking, we'd need to start here and see how this config method resolves the file: https://probot.github.io/api/latest/classes/context.Context.html#config

It also makes it seem like the file should just be in `.github` though.

Reply:
Fair enough 🙂